pandas/tests/frame/test_nonunique_indexes.py

```diff
@@ -320,7 +322,9 @@ def test_dup_columns_across_dtype(self):
 
     def test_set_value_by_index(self, using_array_manager):
         # See gh-12344
-        warn = FutureWarning if using_array_manager else None
+        warn = (
+            FutureWarning if using_array_manager and not is_platform_windows() else None
+        )
```

Comment:
do we know why windows matters here?

Reply:
No, weird ArrayManager thing, but don't have a windows machine, so can't really debug